### DISCUSSION TITLE
[J4] Any plan for improving the admin area design?

### DISCUSSION THREAD

**Discussion by impressionestudio** [joomla/joomla-cms#32307] [Q&A] [answered]
I am sorry for bothering you for an already extensively discussed issue.
I would like to ask if there is a plan to make more design improvements in the admin area?
For example the left menu, the main dashboard and the top dark blue bar need some improvements.

**Comment by dgrammatiko** [comment]
> For example the left menu, the main dashboard and the top dark blue bar need some improvements.

What's wrong with each of them? People cannot really understand the case unless you describe it efficiently, so, saying `the left menu` doesn't really help. What's wrong? Why? how can it be improved? etc...

**Comment by ghost** [comment]
what improvements?

**Comment by impressionestudio** [comment]
There are many other long discussions about the design of the admin area. I don't want to repeat the same things.
For example, as already being said by someone else, the admin area, mostly the parts I mentioned at the top, look like of an older CMS than Joomla 3.
More specifically, all the buttons, the cards in the dashboard etc have rounded corners but the hovers in the left main menu are square.
The same for the big buttons in the site, system and notifications modules in the dashboard.
I see some inconsistencies and I believe that the admin area should look better, more fresh and modern.
Joomla has many and very good developers and I feel safe and quite satisfied from the code they produce.
But there is need to let some designers to do their part. 
It is not only about how something operates, it is also about how it looks.

**Reply by coolcat-creations** [reply]
I think for that cases yo mention, you can submit a PR and let it test - if you can finetune some apperances I am sure that it will be welcomed.

**Comment by ghost** [comment]
impressionestudio attack your image show us your admin area that look better, more fresh and modern

**Comment by chmst** [comment, marked as answer]
To answer the question: no, at the moment there are no plans for changing the backend in J 4.0

**Comment by rinka88** [comment]
It's good when people need improvement. The eyes want to see beauty and style.
On the change in improvement there was a hot discussion in the [Khonsu](https://github.com/joomla-projects/j4adminui/issues/186) branch from the team of joomshaper.
There was beauty in the design of the admin panel, even many asked to implement it instead of Atum.
But the leadership did not want to accept it. And Khonsu's team closed the project for the time being.
Even if I apply the styles of [this](https://delowar.gitbook.io/joomla-elements/), I think it gets a little better.

**Reply by rinka88** [reply]
Even if a [custom icon](https://github.com/joomla-projects/iconfont) for joomla 4 was invented
